gitrevise/merge.py

```diff
@@ -175,6 +178,67 @@ def merge_blobs(
     repo = current.repo
 
     tmpdir = repo.get_tempdir()
+
+    annotated_labels = (
+        f"{path} (new parent): {labels[0]}",
+        f"{path} (old parent): {labels[1]}",
+        f"{path} (current): {labels[2]}",
+    )
+    (is_clean_merge, merged) = merge_files(
+        annotated_labels, current, base, other, tmpdir
+    )
+
+    if is_clean_merge:
+        # No conflicts.
+        return Blob(repo, merged)
+
+    # At this point, we know that there are merge conflicts to resolve.
+    # Prompt to try and trigger manual resolution.
+    print(f"Conflict applying '{labels[2]}'")
+    print(f"  Path: '{path}'")
+
+    preimage = merged
+    (normalized_preimage, conflict_id, merged_blob) = replay_recorded_resolution(
+        repo, tmpdir, preimage
+    )
+    if merged_blob is not None:
+        return merged_blob
+
+    if input("  Edit conflicted file? (Y/n) ").lower() == "n":
+        raise MergeConflict("user aborted")  # pylint: disable=W0707
```

Comment:
You shouldn't need the `pylint` comment here anymore, as this is no longer in an `except` block